astromodels/functions/functions.py

```diff
@@ -1367,3 +1371,313 @@ def _set_units(self, x_unit, y_unit):
 
     def evaluate(self, x, K, xc):
         return K * np.exp(np.divide(x, -xc))
+
+class DMFitFunction(Function1D):
```

Comment:
This is a very specialized function, useful only for DM studies. I am going to put it in its own file after merging, so we can better keep track of it.